bin/lsmb-request.pl

```diff
@@ -72,6 +74,8 @@ sub call_script {
   my $request = shift @_;
   my $locale = shift @_;
 
+  my $trace = ();
```

Comment:
New variable, but not used?

Same for the additional newlines: file isn't actually changed?